.github/workflows/mkdocs-release.yml

```diff
@@ -0,0 +1,39 @@
+name: mkdocs-release
+
+on:
+  push:
+    branches: [ branch-* ]
+
+jobs:
+  publish-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Checkout hopsworks-tutorials
+        uses: actions/checkout@v3
+        with:
+          repository: logicalclocks/hopsworks-tutorials
+          path: docs/hopsworks-tutorials
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: install deps
+        run: pip3 install pip==22.0.3 && pip3 install 'git+https://github.com/logicalclocks/feature-store-api@master#egg=hsfs[docs]&subdirectory=python'
+
+      - name: copy tutorial images
+        run: python prepare_images.py
+
+      - name: setup git
+        run: |
+          git config --global user.name Mike
+          git config --global user.email mike@docs.hopsworks.ai
+
+      - name: mike deploy docs
```

Comment:
We could remove this part from the release script on master. Otherwise when someone makes branch-3.1 it will publish it as 3.0 unless we forget to also bump the version here.